src/darker/tests/test_verification.py

```diff
@@ -28,6 +33,29 @@ def test_verify_ast_unchanged(dst_content, expect):
         assert expect is None
 
 
+def test_ast_verifier_is_equivalent():
+    """``darker.verification.ASTVerifier.is_equivalent_to_baseline``"""
+    verifier = ASTVerifier(baseline=TextDocument.from_lines(["if True: pass"]))
+    assert verifier.is_equivalent_to_baseline(
+        TextDocument.from_lines(["if True:", "    pass"])
+    )
+    assert not verifier.is_equivalent_to_baseline(
+        TextDocument.from_lines(["if False: pass"])
+    )
+
+
+def test_ast_verifier_assert_equivalent():
```

Comment:
This test can go if the unused `assert_equivalent_to_baseline()` method is removed.

Reply:
See my comment about backwards compatibility of importable API.